src/aat/resources/features/F-032/F-032.feature

```diff
@@ -0,0 +1,57 @@
+@F-032
+Feature: F-032: Get printable documents for a case
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-124
+  Scenario: must retrieve printable documents successfully for correct inputs
+    Given a case that has just been created as in [F-032_Case_Creation_Data_With_Document]
+    And a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains the case data of the case just created]
```

Comment:
endpoint: /callback/jurisdictions/{jid}/case-types/{ctid}/documents
No need to use the case in the request.

Reply:
the case data is given in the body of the request